pyggi/base/program.py

```diff
@@ -20,13 +20,12 @@
 from .. import PYGGI_DIR
 from ..utils import Logger, weighted_choice
 
-class StatusCode(enum.Enum):
-    NORMAL = 'NORMAL'
-    TIME_OUT = 'TIME_OUT'
-    PARSE_ERROR = 'PARSE_ERR'
-
-class ParseError(Exception):
-    pass
+class RunResult:
+    def __init__(self, status='ERROR', fitness=None):
```

Comment:
Is there any reason that you set the default status to `ERROR`? Just wondering.

Reply:
Basically just to make sure that it is changed later.
It can be changed to `UNKNOWN` or nothing (`__init__(self, status, fitness=None)`) if you prefer.

Reply:
I prefer `__init__(self, status, fitness=None)` so that the status should be explicitly specified.

Reply:
No problem, I amended the previous change.